### DISCUSSION TITLE
Upload error

### DISCUSSION THREAD

**Discussion by Sorgaz** [iterate-ch/cyberduck#15758] [Q&A]
Hi everyone

For some time I have found a file uploading problem with Smartsync.

Mountain Duck loads one file at a time instead of the classic 2GB package with percentage progression, also feedback that there is no loading of actual files but the continuous loading of the same folders.

I tried to delete the uploads as per the guide on the official site but the problem persists.

I ask you for help!

Thank you

**Comment by tatze96** [comment]
- Which version of Mountain Duck is in use?
- Does this issue occur in combination with a specific protocol?

Please send us the logging output with enabled [debug logging](https://docs.cyberduck.io/mountainduck/support/#debug-log) right after the issue occurs again to support@mountianduck.io with the ticket reference GH-15758 for further investigation.
